style/schedule.css

```
@@ -26,14 +59,53 @@
 }
 
 .schedule-table th, td {
-  padding: 12px;
+  padding: 15px;
   text-align: center;
+  font-size: 18px;
 }
 
 .schedule-table thead {
-  background-color: #93BE37;
+  background-color: black;
 }
 
 .schedule-table tbody tr:nth-child(even) {
-  background-color: #e7e7e7;
+  background-color: var(--color-d);
+}
+
+
+.table-link:hover {
+  font-size: 1.2rem;
 }
+
+.table-link:hover ~ hr {
+  width: 78px;
+}
+
+hr {
+  width: 0px;
+  height: 3px;
+  background: var(--underline);
+  border: var(--underline);
+  margin: auto;
+  transition: width 450ms ease-out;
+}
+
+.table-link {
+  text-decoration: none;
+  color: azure;
+  transition: all 200ms ease-out;
+}
+
+.highlight-text {
+  font-weight: bolder;
+  color: black;
+}
+
+@media only screen and (max-width: 600px) {
+  .schedule-gear {
+    width: 82px;
+  }
+  .container {
+    padding-top: 115px;
+  }
+}
```

Just a crlf issue at the bottom of the file :) 
```suggestion
}

```